Chore/sce lce pce (#9)
loanmarket/career-ladder: Cloud.md

- I am learning about security concerns related to cloud computing and actively seeking how to mitigate any risks.
- I am learning to be competent with the tools and core building blocks used by our team.

> - I am learning how to write scripts to support the deployment of solutions provided by the development team

Reviewer: ```suggestion
- I am learning how to write scripts to support the deployment of solutions provided by the development team.
```

#### I am emerging as a driver for technical and business opportunities.

- I am looking for new technologies and practices that maybe able to assist our team to achieve their goals and improve how we work.
- I am comfortable with building a product vision with a Product or Engineering Manager based on the business needs, regardless of whether this involves technology or otherwise.
- I know we need to be able to measure performance and progress in my team. I am constantly looking for ways to understand what we have achieved.

#### I am investing back into Loan Market.

- I am becoming a trusted voice in Loan Market and my opinions and ideas seem to carry some weight.
- I am mentoring another team member, deliberately helping them to round out their skill set.
- I am sharing my on-the-job learning and experiences with others so they can be more effective in their roles.

---

## Cloud Engineering Manager
> I am trusted to autonomously lead people through any engagement with a proven record of delivering pitches from inception to their successful completion.

> >

Reviewer: This line can go?
```suggestion
```

- I am prioritising the needs of my team members above my own, knowing that their success is our success.
- I am an outspoken advocate of my team. They can trust me to be the first line of defence.
- I know the buck stops with me. At the end of the day I am responsible for my team and the relationship with our peers and customers.
- I actively track the progress of our team, keeping the goals of our projects and our constraints front of mind. I make this information visible and transparent internally and externally to minimise 'surprises' and to facilitate early and pragmatic problem resolution.
- I have a track record of delivering successful projects by applying good practices when leading teams.
- I don't mind making the coffee or picking up the pizza for everyone in my team.
- I am focussed on helping my team grow, sharing what I have learned and encouraging others to do the same.
- I actively delegate tasks to my team providing them autonomy to complete the task. I trust my team to complete tasks and seek help when needed.
- I understand culture is held in our people. I am proactive in hiring for good culture fit and attitude as much as skills.

> #### I am a driver for technical and business opportunities.

Reviewer: Probably a blank line before this?

Author: updated.

- I possess writing skills that let me be clear, concise, articulate and persuasive in all of my outgoing communications.

#### I've become a student of my craft.

- I am devoted to learning; it's become a natural part of my life.
- I am actively introspective and take time to work on my personal and professional development.
- I happily take responsibility for my own mistakes and I learn from the mistakes of others.

---

## Principal Cloud Engineer
> I am a well-rounded cloud developer, architect, problem solver and leader of people. I aspire to have engineering and behavioural excellence.
>
> 🔀 This role starts the Domain Expert track. 

#### I am trusted to lead a team and take an idea all the way to production with little supervision.

- I am emerging as a leader of people.
- I can confidently step outside my comfort zone and adapt quickly to new team situations.
- I am comfortable diving head-first into moderately risky environments with relatively little upfront information and oversight.
- I am comfortable with transparently assessing risk, making recommendations, escalating appropriately and dealing with the consequences along the way.

> - I am proficient at leading software projects using agreed practices.

Reviewer: ```suggestion
- I am proficient at leading projects using agreed practices.
```

- I can confidently and charismatically pitch ideas and influence my teams decisions.
- I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.

#### I am learning to be a servant leader for my team.

- I am prioritising the needs of my team members above my own, knowing that their success is our success.
- I am learning to reliably deliver successful projects by applying good practices when leading teams.
- I am focussed on helping my peers grow, sharing what I have learned and encouraging others to do the same.
- I facilitate ongoing, net-positive and productive communication in my team.
- I participate and have a strong voice in the hiring of new team members.
- I understand that I cannot solve every problem for my team by doing it myself. I am learning to trust and delegate tasks to my team.

#### I am emerging as a driver for technical and business opportunities.

- I am looking for new technologies and practices that maybe able to assist our team to achieve their goals and improve how we work.

> - I am comfortable with building a product vision with an Engineering Manager based on the business needs, regardless of whether this involves technology or otherwise.

Reviewer: ```suggestion
- I am comfortable with building a product vision with a Product or Engineering Manager based on the business needs, regardless of whether this involves technology or otherwise.
```

- I know we need to be able to measure performance and progress in my team. I am constantly looking for ways to understand what we have achieved.

#### I am investing back into Loan Market.

- I am becoming a trusted voice in Loan Market and my opinions and ideas seem to carry some weight.
- I am mentoring another team member, deliberately helping them to round out their skill set.
- I am sharing my on-the-job learning and experiences with others so they can be more effective in their roles.

---

## Cloud Engineering Manager
> I am trusted to autonomously lead people through any engagement with a proven record of delivering pitches from inception to their successful completion.
>

> > 🔀 The sub-branches of Engineering Manager are:

Reviewer: do these sub-branches apply? Is cloud engineering a place where you can lead effectively without being an "architect"?

Author: updated now.

- I am prioritising the needs of my team members above my own, knowing that their success is our success.
- I am an outspoken advocate of my team. They can trust me to be the first line of defence.
- I know the buck stops with me. At the end of the day I am responsible for my team and the relationship with our peers and customers.
- I actively track the progress of our team, keeping the goals of our projects and our constraints front of mind. I make this information visible and transparent internally and externally to minimise 'surprises' and to facilitate early and pragmatic problem resolution.
- I have a track record of delivering successful projects by applying good practices when leading teams.
- I don't mind making the coffee or picking up the pizza for everyone in my team.
- I am focussed on helping my team grow, sharing what I have learned and encouraging others to do the same.
- I actively delegate tasks to my team providing them autonomy to complete the task. I trust my team to complete tasks and seek help when needed.
- I understand culture is held in our people. I am proactive in hiring for good culture fit and attitude as much as skills.

> - My validation and success comes from the cumulative successes of my team.

Reviewer: Feel free to ignore me but my preference would be to not have this statement. I think I get your sentiment but I fear others may read it differently to what you intended. I would hope the other examples would show they are recognised for how their team achieves rather than themselves.

Author: updated and removed.

#### I am a driver for technical and business opportunities.

- I am looking for new opportunities and thinking beyond what we are creating today.

> - I don't rely on the Product Manager to define the roadmap and set objectives. I am constantly asking questions and looking for ways to improve our intentions.

Reviewer: ```suggestion
- I don't rely on the Product and Engineering Managers to define the roadmap and set objectives. I am constantly asking questions and looking for ways to improve our intentions.
```

- I listen to my team but advocate for more. I look to change negative attitudes into positive plans of action.

#### I am investing back into Loan Market.

- I am a trusted voice in Loan Market and my opinions and ideas carry some weight.
- I am helping steer the public and private conversations within the business towards positive and productive outcomes.
- I represent the interest of my team and the technology group as a whole to the wider business.

> - I am involved in initiatives beyond developing software.

Reviewer: ```suggestion
- I am involved in initiatives beyond cloud engineering.
```
